internal/storage/entry.go

```diff
@@ -54,11 +54,6 @@ func (entry Entry) Validate() (bool, []error) {
 }
 
 func (entry Entry) DeDupe() {
-	type fileCounter struct {
```

Comment:
another one -is it not necessary to declare the type? I noticed further down that there was a reference to this slice on line 79...again my ignorance of the language, probably...

Reply:
There is already a definition for this type [further up the file](https://github.com/ministryofjustice/opg-file-service/pull/20/files/a2931d775da8870cf363dd898cfa1b7559667f89#diff-d822bdefef4fd1e6230d6db2174ad28fR16). But also it is kind of weird to define types _inside_ functions, they should live at the top-level.

Reply:
sorry missed that one! agree not the right place, at least from a structural point of view.